data/posts/react-router-v7.md

```diff
@@ -0,0 +1,48 @@
+---
+title: React Router v7
+summary: React Router v7 brings all the great things you love from Remix back to React Router
+date: 2024-11-22
+image: /blog-images/headers/react-router-v7.jpg
+ogImage: /blog-images/headers/react-router-v7.jpg
```

Comment:
Whoops, one more. Forgot I added this option, really should have documented this stuff

```suggestion
ogImage: /blog-images/headers/react-router-v7.jpg
imageDisableOverlay: true
```

Reply:
makes it look like:

<img width="980" alt="image" src="https://github.com/user-attachments/assets/1bd4412f-402d-4923-a670-93ad7e315cce">

instead of

<img width="888" alt="Screenshot 2024-11-22 at 4 06 09 PM" src="https://github.com/user-attachments/assets/831e4cb9-ab03-496d-aaff-298269d16a2b">